.phpcs.xml.dist

```diff
@@ -112,7 +112,7 @@
 	</rule>
 
 	<!-- Include sniffs for PHP cross-version compatibility. -->
-	<config name="testVersion" value="5.6-"/>
+	<config name="testVersion" value="7.0-"/>
```

Comment:
The plugin requires PHP 7+

Without this we're getting yelled at by PHPCS for using return type declarations.

Reply:
Oh, good catch. This should have been updated as part of #7088.